### ISSUE TITLE
[WEBAPP] Validation in onboarding pages

### ISSUE THREAD

**Issue by ppagarawal** [FightPandemics/FightPandemics#506]
Hello Team,

Please note that the onboarding pages must activate the next icon only when the input is provided and valid. The validation rules need to be followed for both Give Help/Take Help workflows

Validation rules:
Question 1/3
The next button gets activated only after selection of either "Medical" or 'Other Help"

Question 2/3
Location must be from the dropdown/detected and valid for the next button to be active

![image](https://user-images.githubusercontent.com/6276256/83173031-0d336780-a136-11ea-88be-43fd9d87538b.png)

![image](https://user-images.githubusercontent.com/6276256/83173071-1d4b4700-a136-11ea-93ce-8e9599be46ea.png)

Question 3/3
User must enter a valid email address to proceed. 

If the validation conditions are not met, the next button should not be activated.

Feel free to reach out to me in case of questions.

**Comment by snowiesuet** [comment]
I think @Yu-Alexa-Zhao mentioned that these are not mandatory fields. Please confirm, thanks!

**Comment by ppagarawal** [comment]
@snowiesuet I understand that these are not mandatory at the moment however, I think that the input needs to be valid. By valid, I mean that the data needs to make sense and cannot be non-realworld data.

**Comment by Yu-Alexa-Zhao** [comment]
Yes, this is not a mandatory field but still needs validation I think

**Comment by ppagarawal** [comment]
@snowiesuet I think that you need to validate the email address in the third step only and then display "Invalid email address" if the email is invalid. Ignore step 1 and step 2 for validation since the data will be populated from the dropdown or given list.

**Comment by snowiesuet** [comment]
Got it! Thanks!
